app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,70 @@
+require 'date'
```

Comment:
Even though we're using the `DateTime` class later on in this file, we actually do not need to require the date library here.

In general, because Rails is fancy, we do not need to explicitly `require` any libraries that we want to use. For date and time related classes Rails provides us with several custom ones that build on top of the default Ruby classes. And for any gems that we will use Rails will actually recognize that we've included them in our `Gemfile` and require the necessary files for us.